src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String newLine = System.lineSeparator();
+    private final StringBuilder builder = new StringBuilder();
```

Comment:
You don`t need StringBuilder as field of calss, System.lineSeparator() can be constatnt